### PR TITLE
Config/app/new/mock/stacks/run

### PR DESCRIPTION
**Pull request by talhainvenxion**
Stack functionality is complete.

### CODE REVIEW

File: src/api/mockApiData.ts [hunk 1]
```diff
@@ -16,116 +16,211 @@ const mockApi = {
...
-        training_args: {},
-        serving_args: {},
+        id: 'asdf2-asdf1-ad143',
+        type: 'alerter',
+        flavor: 'slack',
+        name: 'slack_alerter',
+        config: {
+          slack_token: 'INFU1328UJ',
+          default_slack_channel_id: 184195,
+        },
```
Comment (danialraza03) [inline src/api/mockApiData.ts:185]: This is the old stack response structure, just want to confirm that we are not using it now?

File: src/api/mockApiData.ts [hunk 2]
```diff
@@ -16,116 +16,211 @@ const mockApi = {
...
-          training_args: {},
-          serving_args: {},
+          id: 'asdf2-asdf1-ad143',
+          type: 'alerter',
+          flavor: 'slack',
+          name: 'slack_alerter',
+          config: {
+            slack_token: 'INFU1328UJ',
+            default_slack_channel_id: 184195,
+          },
```
Comment (danialraza03) [inline src/api/mockApiData.ts:32]: Old stack response structure

File: global.d.ts [hunk 3]
```diff
@@ -89,6 +92,11 @@ interface TRun {
   workspaceId: TId;
```
Comment (danialraza03) [inline global.d.ts:92]: WorkSpace Id to be converted to project name, or?

Comment (talhainvenxion) [reply global.d.ts:92]: For now, workSpaceId is used in many places like old pipelines and runs that's why we need it as a type in typescript.

**Pull request merged**